protocol/0028-GOVE-governance.md

```diff
@@ -131,6 +131,27 @@ If a proposal is created and later a different outcome is preferred by network p
 
 Which of these makes most sense will depend on the type of change, the timing of the events, and how the rest of the community votes for the initial proposal.
 
+## Cancelling a proposal
+
+A proposal can be cancelled under certain conditions by, and only by, the party which created the proposal. To avoid situations where the community may have acted upon proposals they "expected" to pass, a party can only cancel their proposal before a certain "cancellation threshold" is reached.
+
+This cancellation threshold is a fixed point before the proposal closing time. The duration of the period in which a proposal cannot be cancelled is defined by the network parameter `governance.proposal.cancellationThreshold` (a duration string defaulting to `24h0m0s`).
+
+i.e. Given the `governance.proposal.cancellationThreshold` is `24h0m0s` a party will be able to cancel their proposal up until 1 day before the proposal closing time.
+
+### Mechanics
+
+A party can cancel their proposal by simply voting against it with any vote value.
```

Comment:
What does "with any vote value" mean here? I think votes are VALUE_YES and VALUE_NO so you would need to vote VALUE_NO?